misc/docs/source/tutorial/code_snippets/basemap_plot_custom.py

```diff
@@ -0,0 +1,37 @@
+from mpl_toolkits.basemap import Basemap
+import numpy as np
+import matplotlib.pyplot as plt
+
+from obspy import read_inventory, read_events
+
+# Set up a custom basemap, example is taken from basemap users' manual
+
+# setup albers equal area conic basemap
+# lat_1 is first standard parallel.
+# lat_2 is second standard parallel.
+# lon_0, lat_0 is central point.
+m = Basemap(width=8000000, height=7000000,
+            resolution='l', projection='aea',
+            lat_1=40., lat_2=60, lon_0=35, lat_0=50)
+m.drawcoastlines()
+m.drawcountries()
+m.fillcontinents(color='coral', lake_color='aqua')
+# draw parallels and meridians.
+m.drawparallels(np.arange(-80., 81., 20.))
+m.drawmeridians(np.arange(-180., 181., 20.))
+m.drawmapboundary(fill_color='aqua')
+# draw tissot's indicatrix to show distortion.
```

Comment:
Oh, thanks, that's a part I removed from the original basemap example.. overlooked to remove the comment as well..